src/parsers/power_models_data.jl

```diff
@@ -1079,8 +1079,10 @@ function make_branch(
     primary_shunt = d["b_fr"]
     alpha = d["shift"]
     branch_type = get_branch_type(d["tap"], alpha, d["transformer"])
-
-    if d["transformer"]
+    if d["br_r"] == 0.0 && d["br_x"] == 0.0
```

Comment:
Have you included the option of checking for status to set it correctly to open or closed?

Reply:
No, right now I only match the `availability`. Are you thinking we should implement this ?
Line w/ zero impedance + `available = true` -> DiscreteControlledACBranch  w/ `available = true` and `status = closed`
Line w/ zero impedance + `available = false` -> DiscreteControlledACBranch  w/ `available = false` and `status = open`

Reply:
This makes sense, we should record this in a log entry.